src/tests/cases/ui.rs

```diff
@@ -112,6 +112,115 @@ fn pause_by_space() {
     assert_snapshot!(&terminal_draw_events_mirror[2]);
 }
 
+#[test]
+fn basic_only_processes() {
```

Comment:
These tests are great! Let's add some data to them as well, ok? Just one additional snapshot per test with data. What do you think?